docs/docs/configuration/animations.mdx

```diff
@@ -0,0 +1,224 @@
+---
+title: Animations
+---
+
+Chart.js animates charts out of the box. A number of options are provided to configure how the animation looks and how long it takes.
+
+import { useEffect } from 'react';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Tabs
+    defaultValue='tension'
+    values={[
+        {label: 'Looping tension', value: 'tension' },
```

Comment:
Maybe we should call this `Looping tension [property]` and "Hide and show [mode]" or something to help link it to the docs below and make it clearer what it's demonstrating? We could also maybe say in the table something like "See the example above for a demonstration" for each of those two properties

Reply:
done